### PR TITLE
2.8.19 release secfixes

### PR DESCRIPTION
**Pull request by uncleflo**
This is an upgrade merge request, following https://github.com/centreon/centreon/pull/6049 and to fulfill a conflict resolution request.

I have remerged secfixes with the latest available release, but could not recreate the conflicts regarding split to explode function changes. 

If this issue is still ongoing, please resolve these conflicts yourself, as there are 3 or 4 code-lines changed only. I have no idea WHICH split calls you changed, and I think its not a could idea for me to GUESS which split functions you have already changed.

### CODE REVIEW

File: www/class/centreonEscaping.php [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+ * A class for convenience. 
+ * This class makes short code style possible using Esc::forHtmlAttr(). 
+ * 
+ * This is prefered as opposed to CentreonEscape::forHtmlAttr(), which 
+ * can be long and unclear, unoverseeable in lines with more than 3 values to 
+ * escape, and hence prone to security mistakes. This class makes it clearer 
+ * for developers' understanding and analogous with verbal understanding 
+ * eg. "Escape (this value) for Html Attributes":  Esc::forHtmlAttr($sThisValue)
+ */
+class Esc extends CentreonEscape { }
```
Comment (loiclau) [inline www/class/centreonEscaping.php]: hi, can you make a file by class, and name the file like the class.

**Comment by uncleflo** [comment]
Done.

To note: I think its a bad idea to go away from the `centreon*.class.php` convention, but you did request it.

File: www/class/centreonEscaping.php (hunk 1 above)
Comment (uncleflo) [reply www/class/centreonEscaping.php]: See comment below, the request has been done

**Comment by lpinsivy** [comment]
Can you remove documentation (release note) part ?

**Comment by uncleflo** [comment]
I will be creating a new pull request to accomodate the removal of the centreon-2.8.19.rst note

**Pull request closed without merging**